src/manifest.ts

```diff
@@ -15,7 +15,7 @@ export const DEFAULT_DEVICE = 'deviceName';
 export const CONFIG_PROCESSOR = 'processorConfig';
 export const DEFAULT_PROCESSOR = 'processorName';
 export const CONFIG_ADDRGAP = 'svdAddrGapThreshold';
-export const DEFAULT_ADDRGAP = 16;
+export const DEFAULT_ADDRGAP = -1;
```

Comment:
Setting to `-1` is useful for testing purposes as discussed "offline". But we should not change the default on `main`.